### PR TITLE
feat(STN-782) Limited width color bands

### PR DESCRIPTION
**Pull request by rkelbel**
# READY FOR REVIEW

## Summary
As a Developer, Site Owner, or Contributor, I want a limited width option for Color Band that only spans the center column.

## Need Review By (Date)
asap

## Urgency
medium

## Steps to Test
1. npm run test
2. import the config
3. create a flexible page with color bands, limited and full width
4. Make sure both options work as expected

## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Sparkbox PR Checklist](../docs/SparkboxPRChecklist.md)

### CODE REVIEW

File: docroot/themes/humsci/humsci_basic/src/scss/components/_pattern.color-band.scss [hunk 1]
```diff
@@ -2,26 +2,28 @@
...
-    content: '';
-    display: block;
-    width: 50vw;
-    position: absolute;
-    top: 0;
-    bottom: 0;
-
-    .layout-builder & {
-      display: none;
+  &--full-width{
```
Comment (heatherdesigns) [inline docroot/themes/humsci/humsci_basic/src/scss/components/_pattern.color-band.scss]: Let's add a space between the `h` and `{` to make stylelint happy. 😄

**Review by heatherdesigns — COMMENTED** [review]
This looks fantastic! 

well done

There is one small update to the CSS and it is ready to roll.

I used three pages for testing. The first page was a pre-existing color band page. I wasn't sure what to expect but wanted to notate that all the bands on the page were the new default style using the limited-width. I know you're working with Mike on a pre-process hook. I'm not quite sure how to test that.

I also tested limited and full width color bands on a page with no sidebars and on a page with two sidebars. Everything worked as expected and both pages looked great! :tada:

I did notice that the `- None -` option was available in the width dropdown. Is that something we need to remove?
none

File: docroot/themes/humsci/humsci_basic/src/scss/components/_pattern.color-band.scss (hunk 1 above)
Comment (rkelbel) [reply docroot/themes/humsci/humsci_basic/src/scss/components/_pattern.color-band.scss]: This should be fixed up!

**Review by heatherdesigns — APPROVED** [review]
Excellent work!!!

celebrate

**Pull request merged**